charts/kube-prometheus-stack/README.md

```diff
@@ -196,9 +196,12 @@ For more in-depth documentation of configuration options meanings, please see
 
 ## prometheus.io/scrape
 
-The prometheus operator does not support annotation-based discovery of services, using the `serviceMonitor` CRD in its place as it provides far more configuration options. For information on how to use servicemonitors, please see the documentation on the `prometheus-operator/prometheus-operator` documentation here: [Running Exporters](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/running-exporters.md)
+The prometheus operator does not support annotation-based discovery of services, using the `podMonitor` or `serviceMonitor` CRD in its place as they provide far more configuration options. For information on how to use podmonitors and servicemonitors, please see the documentation on the `prometheus-operator/prometheus-operator` documentation here:
```

Comment:
Suggestions:

1. The CRDs should be CamelCased, right?
    - `podMonitor`-> `PodMonitor`
    - `serviceMonitor` -> `ServiceMonitor`
1. "podmonitors and servicemonitors" -> "PodMonitors/ServiceMonitors" for consistency with your other paragraph change below.
1. Added line break, for one sentence per line md style rule (not yet able to add to automated linting. See https://github.com/prometheus-community/helm-charts/issues/25)


```suggestion
The prometheus operator does not support annotation-based discovery of services, using the `PodMonitor` or `ServiceMonitor` CRD in its place as they provide far more configuration options.
For information on how to use podmonitors and servicemonitors, please see the documentation on the `prometheus-operator/prometheus-operator` documentation here:
```

Reply:
Updated my commit suggestion